utils/symbol.c

```diff
@@ -1600,6 +1608,7 @@ TEST_CASE(symbol_load_map) {
 	struct symtabs symtabs = {
 		.dirname = "",
 		.kernel_base = -4096ULL,
+		.flags = SYMTAB_FL_ADJ_OFFSET,
```

Comment:
Basically non-PIE has same issue and I think it works as is.  Can you verify this change doesn't break other arch?

Reply:
@namhyung x86_64 was ok with this. but I did not test it on the Arm base.

Reply:
OK, then let's change it like below:

```c
for (i = 0; i < info->dlpi_phnum; i++) {
    if (info->dlpi_phdr[i].p_type != PT_LOAD)
        continue;
    map->start = info->dlpi_addr + info->dlpi_phdr[i].p_vaddr;
    map->end = map->start + info->dlpi_phdr[i].p_memsz;
    break;
}
```

Reply:
It was applied! test passed at i386 and x64